tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{38,39,310,311,312},pypy
+envlist = py{39,310,311,312},pypy
```

Comment:
We should add "313" to this list as well to make sure we're getting testing against Python 3.13 on Travis.  That'll make for an interesting test to see if our Travis runners can do that... but we should at least start there.